static/app/components/profiling/flamegraphZoomViewMinimap.tsx

```diff
@@ -245,6 +256,27 @@ function FlamegraphZoomViewMinimap({
         setLastInteraction('pan');
         return;
       }
+
+      if (evt.buttons === 1 && lastInteraction === 'resize') {
```

Comment:
Why do we check for evt.buttons here? Are we trying to check that the user is holding the button?

Reply:
Yeah its to determine if mousedown is still being held.  I don't think we need it anymore and will drop it. Pretty sure i threw it in there when i was dealing w/ that issue where i was losing the mouse interaction... at that time i was piggy backing off of the `select` interaction and shot myself in the foot.